Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:9
+# Docker build arguments
+ARG SOURCE_DIR=/repo
+ARG ARTIFACT_DIR=/dist
+ARG ANDROID_DIR=/usr/lib/android-sdk
+# Docker run environment
+ENV SOURCE_DIR=/repo
+ENV ARTIFACT_DIR=/dist
+ENV ANDROID_DIR=/usr/lib/android-sdk
+
+# Prepare Debian build environment
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list \
+ && apt-get update \
+ && apt-get install -y mmv wget unzip git \
+ && apt-get install -t stretch-backports -y android-sdk openjdk-8-jdk
+
+# Install Android repository tools
+RUN rm -rf ${ANDROID_DIR}/tools \
+ && wget http://dl-ssl.google.com/android/repository/tools_r25.2.3-linux.zip -O tools.zip \
+ && unzip tools.zip -d ${ANDROID_DIR}/ \
+ && rm -f tools.zip
+
+# Install SDK tools
+# `yes` is required to accept licenses:
+# https://stackoverflow.com/questions/38096225/automatically-accept-all-sdk-licences/42125740#42125740
+# There will be a bit of delay (approx 30 seconds)
```

Comment:
I know I'm repeating myself but this should be something along the lines of "up to a couple minutes".

Reply:
Agreed, I updated the wording.